garglk/config.cpp

```diff
@@ -38,6 +38,10 @@
 bool gli_utf8input = true;
 bool gli_utf8output = true;
 
+int gli_hires = 1;
```

Comment:
This should be `bool` and `true` now (when these patches were originally done, `bool` didn't exist in the code).

Reply:
Note to self: Build before pushing.